javascript-arrays/main.js

```diff
@@ -0,0 +1,29 @@
+const colors = ['red', 'white', 'blue'];
+
+console.log('Value of colors[0]:', colors[0]);
+console.log('Value of colors[1]:', colors[1]);
+console.log('Value of colors[2]:', colors[2]);
+
+console.log(
+  'America is ' + colors[0] + ', ' + colors[1] + ', ' + 'and ' + colors[2] + '.'
+);
+
+colors[2] = 'green';
+
+console.log(
+  'Mexico is ' + colors[0] + ', ' + colors[1] + ', ' + 'and ' + colors[2] + '.'
+);
+console.log('Value of colors:', colors);
+
+const students = ['Anna', 'Nat', 'Katy', 'Bella'];
+
+const numberOfStudents = students.length;
+
+console.log('There are', numberOfStudents, 'students in the class.');
+
+const lastIndex = students[students.length - 1];
```

Comment:
Instead of doing `students.length`, you should be subtracting 1 from `numberOfStudents`. We already have a variable `numberOfStudents` with the value of `students.length`, so we can just utilize that variable.